elementalconductor/presets.go

```diff
@@ -0,0 +1,70 @@
+package elementalconductor
+
+// GetPresets returns a list of presets
+func (c *Client) GetPresets() (*PresetList, error) {
+	var result *PresetList
+	err := c.do("GET", "/presets", nil, &result)
+	if err != nil {
+		return nil, err
+	}
+	return result, nil
+}
+
+// GetPreset return details of a given presetID
+func (c *Client) GetPreset(presetID string) (*Preset, error) {
+	var result *Preset
+	err := c.do("GET", "/presets/"+presetID, nil, &result)
+	if err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
+
+// CreatePreset creates a new preset
+func (c *Client) CreatePreset(p *Preset) (*Preset, error) {
```

Comment:
I just noticed that @marzagao is using `PostJob` instead of `CreateJob` so maybe I should change this to maintain compatibility